libs/ledger-live-common/src/bridge/jsHelpers.ts

```diff
@@ -81,14 +81,22 @@ export type GetAccountShapeArg0 = {
 };
 
 export type GetAccountShape = (
-  arg0: GetAccountShapeArg0,
-  arg1: SyncConfig
+  infos: AccountShapeInfos,
+  sysConfig: SyncConfig
 ) => Promise<Partial<Account>>;
-type AccountUpdater = (arg0: Account) => Account;
+type AccountUpdater = (account: Account) => Account;
 
+/**
+ * @WARNING
+ * @DEPRECATED due to circular dependency
+ */
 // compare that two dates are roughly the same date in order to update the case it would have drastically changed
 const sameDate = (a, b) => Math.abs(a - b) < 1000 * 60 * 30;
 
+/**
+ * @WARNING
+ * @DEPRECATED due to circular dependency
```

Comment:
Shouldn't we just remove it from this file then ?

Reply:
Discussed with Kevin on slack... a PR should be opened later that removes the copy in jsHelpers and adapts every family